src/kw_include.sh

```diff
@@ -30,6 +30,9 @@ function include()
     return 0
   fi
 
-  declare -g "${varname}"=1
+  ret=$(declare -g "${varname}"=1 2>&1)
```

Comment:
What kind of error have you seen associated with `declare`?

Notice that this code changed the overall behavior of the include function; after your change, we start to have these warnings in the CI:

```
./src/kwio.sh: line 2: declare: BLUECOLOR: readonly variable
./src/kwio.sh: line 3: declare: REDCOLOR: readonly variable
./src/kwio.sh: line 4: declare: YELLOWCOLOR: readonly variable
./src/kwio.sh: line 5: declare: GREENCOLOR: readonly variable
./src/kwio.sh: line 6: declare: SEPARATOR: readonly variable
```
https://github.com/kworkflow/kworkflow/pull/474/checks?check_run_id=3798086709

Tbh, I prefer the original code in this part.